### ISSUE TITLE
def anchor_loss(onehot_labels, logits, weights, gamma=0.5, slack=0.05):

### ISSUE THREAD

**Issue by aoko-sys** [taekwan-lee/AnchorLoss_TF#1]
Hello！
I want to ask, can I add weights to 1?If not, how to get the weights?

**Comment by taekwan-lee** [comment]
@aoko-sys 

Yes, you can use weights with value 1. This way original anchor loss is calculated.
